### DISCUSSION TITLE
Find multiple values on the same column?

### DISCUSSION THREAD

**Discussion by phollyer** [ujjwalguptaofficial/JsStore#260] [Q&A] [answered]
Hi Ujiwal,

First off, thanks for JsStore and IDBStudio, they're both excellent.

The only problem I'm having at the moment is querying for multiple values on the same column. 

I tried the following without success:

```
select({
    from: "Customers",
    where: {
        country: 'Mexico',
        or: {
            city: ['London','Berlin']
        }
    }
});
```

Is it possible?

Many thanks.

**Comment by ujjwalguptaofficial** [comment, marked as answer]
yes its possible. You need to use `in` for the array query.

```
select({
    from: "Customers",
    where: {
        country: 'Mexico',
        or: {
            city: {
                in: ['London', 'Madrid']
            }
        }
    }
});
```

Here is idbstudio link - https://ujjwalguptaofficial.github.io/idbstudio/?db=Demo&query=select(%7B%0A%20%20%20%20from%3A%20%22Customers%22%2C%0A%20%20%20%20where%3A%20%7B%0A%20%20%20%20%20%20%20%20country%3A%20'Mexico'%2C%0A%20%20%20%20%20%20%20%20or%3A%20%7B%0A%20%20%20%20%20%20%20%20%20%20%20%20city%3A%20%7B%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20in%3A%20%5B'London'%2C%20'Madrid'%5D%0A%20%20%20%20%20%20%20%20%20%20%20%20%7D%0A%20%20%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%7D)%3B

**Reply by phollyer** [reply]
Many thanks.
